src/ivoc/ocmatrix.h

```diff
@@ -24,52 +25,63 @@ class OcMatrix {
         unimp();
         return nullptr;
     }  // matrix element pointer
+
+    inline double operator()(int i, int j) const {
+        return getval(i, j);
+    };
+
     inline double& operator()(int i, int j) {
         return *mep(i, j);
     };
 
-    virtual double getval(int i, int j) {
+    inline double& coeff(int i, int j) {
+        return *mep(i, j);
+    }
```

Comment:
why both `coeff` and `operator()`? if we need both can they be implemented by one calling the other?

Reply:
Sometimes we get the matrix from a pointer and so it is "boring" to call:
`m->operator()(i, j)`. And so I create this `coeff()` to avoid using the bad `mep()`.

I consider `mep()` bad, because `nrn` in the past were storing pointer and keeping them, was working well with `sparse13`, does not work as intended by `Eigen`. So I want to remove `mep()` so people are not "pushed" to keep pointer.

Reply:
you'd do `(*m)(i, j)`. maybe implement `coeff` as `return (*this)(i, j);` then it's at least clear from the implementation that it's meant to be the same.

Reply:
I propose a new way, tell me what you think